### PR TITLE
AMBARI-24895 - Infra Manager: code clean up

### PR DESCRIPTION
**Pull request by kasakrisz**
## What changes were proposed in this pull request?

- rename LOG -> logger
- remove slf4j use log4j2
- merge Job Properties and Job Parameter classes

## How was this patch tested?

ITs passed

Manually:
1. Deploy Ambari and a cluster: zookeeper, infra-solr, logsearch, hdfs, infra-manager
2. Kerberize the cluster
3. Enable archiving for service_logs and set the destination to hdfs on Ambari UI, Infra Manager config
4. Run archiving of service_logs using infra managers rest api
5. check hdfs for archived service logs

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Ambari-Infra-Github-PR-Builder/22/
Test PASSed.

File: ambari-infra-manager/src/main/java/org/apache/ambari/infra/conf/security/HadoopCredentialStore.java [hunk 1]
```diff
@@ -18,16 +18,16 @@
...
 import java.util.Optional;
 
-import static org.apache.commons.lang.StringUtils.isBlank;
-import static org.apache.commons.lang3.ArrayUtils.isNotEmpty;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class HadoopCredentialStore implements PasswordStore {
-  private static final Logger LOG = LoggerFactory.getLogger(InfraManagerSecurityConfig.class);
+  private static final Logger logger = LogManager.getLogger(InfraManagerSecurityConfig.class);
```
Comment (zeroflag) [inline ambari-infra-manager/src/main/java/org/apache/ambari/infra/conf/security/HadoopCredentialStore.java]: what's the difference between LogManager.getLogger and LoggerFactory.getLogger (used at other places)?

File: ambari-infra-manager-it/src/test/java/org/apache/ambari/infra/InfraClient.java [hunk 2]
```diff
@@ -49,7 +49,7 @@
 
 // TODO: use swagger
 public class InfraClient implements AutoCloseable {
-  private static final Logger LOG = LoggerFactory.getLogger(InfraClient.class);
+  private static final Logger logger = LoggerFactory.getLogger(InfraClient.class);
```
[the remark this answers is not in the document]
Comment (oleewere) [reply ambari-infra-manager-it/src/test/java/org/apache/ambari/infra/InfraClient.java]: it is not a constant

File: ambari-infra-manager-it/src/test/java/org/apache/ambari/infra/Solr.java [hunk 3]
```diff
@@ -41,7 +41,7 @@
 import org.slf4j.LoggerFactory;
 
 public class Solr {
-  private static final Logger LOG = LoggerFactory.getLogger(Solr.class);
+  private static final Logger logger = LoggerFactory.getLogger(Solr.class);
```
[the remark this answers is not in the document]
Comment (kasakrisz) [reply ambari-infra-manager-it/src/test/java/org/apache/ambari/infra/Solr.java]: I am going to replace it to make it consistent.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Ambari-Infra-Github-PR-Builder/23/
Test PASSed.

File: ambari-infra-manager-it/src/test/java/org/apache/ambari/infra/InfraClient.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (adoroszlai) [reply ambari-infra-manager-it/src/test/java/org/apache/ambari/infra/InfraClient.java]: @oleewere what's a constant?

File: ambari-infra-manager-it/src/test/java/org/apache/ambari/infra/Solr.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (adoroszlai) [reply ambari-infra-manager-it/src/test/java/org/apache/ambari/infra/Solr.java]: Thanks @kasakrisz!

File: ambari-infra-manager-it/src/test/java/org/apache/ambari/infra/InfraClient.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (oleewere) [reply ambari-infra-manager-it/src/test/java/org/apache/ambari/infra/InfraClient.java]: @adoroszlai what the compiler thinks it is (https://docs.oracle.com/javase/specs/jls/se7/html/jls-15.html#jls-15.28), primitives/arrays or classes/objects where the state cannot change anyhow. object immutability is not enough here, like Logger is an interface. but another opposite example that can be constant is a list (created from a static method), which has final add/remove methods. also worth to check what you can use in annotations.

**Pull request merged**